xml/src/main/resources/ehcache-core.xsd

```diff
@@ -211,6 +211,43 @@
           </xs:sequence>
         </xs:complexType>
       </xs:element>
+        <xs:element name="listener" minOccurs="0" maxOccurs="unbounded">
+          <xs:complexType>
+            <xs:sequence>
+              <xs:element name="class" type="xs:string" minOccurs="1" maxOccurs="1">
+                <xs:annotation>
+                  <xs:documentation xml:lang="en">
+                    A fully qualified class name to a concrete type that implements
+                    org.ehcache.spi.event.CacheEventListener and has a public
+                    default no argument constructor.
+                  </xs:documentation>
+                </xs:annotation>
+              </xs:element>
+              <xs:element name="eventFiringMode" type="ehcache:EventFiring" minOccurs="1" maxOccurs="1">
```

Comment:
Please respect the XSD types naming convention. Type here should be called "ehcache:eventFiring-type"